src/pane.coffee

```diff
@@ -457,13 +457,20 @@ class Pane extends Model
     else
       return true
 
+    currentActiveItem = @activeItem
+    @activateItem item
```

Comment:
This is the only line which should be merged to sublime text like functionality.